tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/TraceAnnotations/RuntimeHandleTuple.cs

```diff
@@ -0,0 +1,69 @@
+// <copyright file="RuntimeHandleTuple.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.TraceAnnotations
+{
+    internal readonly struct RuntimeHandleTuple : IEquatable<RuntimeHandleTuple>
+    {
+        /// <summary>
+        /// The method handle
+        /// </summary>
+        public readonly RuntimeMethodHandle MethodHandle;
+
+        /// <summary>
+        /// The type handle
+        /// </summary>
+        public readonly RuntimeTypeHandle TypeHandle;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RuntimeHandleTuple"/> struct.
+        /// </summary>
+        /// <param name="methodHandle">The method handle</param>
+        /// <param name="typeHandle">The owning type handle</param>
+        public RuntimeHandleTuple(RuntimeMethodHandle methodHandle, RuntimeTypeHandle typeHandle)
+        {
+            MethodHandle = methodHandle;
+            TypeHandle = typeHandle;
+        }
+
+        /// <summary>
+        /// Gets the struct hashcode
+        /// </summary>
+        /// <returns>Hashcode</returns>
+        public override int GetHashCode()
+        {
+            unchecked
+            {
+                var hash = (int)2166136261;
+                hash = (hash ^ MethodHandle.GetHashCode()) * 16777619;
+                hash = (hash ^ TypeHandle.GetHashCode()) * 16777619;
+                return hash;
```

Comment:
Whoa, I thought this would need to be behind `#if NETCOREAPP3_1_OR_HIGHER` or something, but it looks like you're now using the one from https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/src/Datadog.Trace/Vendors/MessagePack/HashCode.cs, which is fine.